src/app/api/schemas.py

```diff
@@ -163,8 +163,8 @@ class SoftwareHash(BaseModel):
 class MyDeviceIn(Login, DefaultPosition):
     specs: str = Field(..., min_length=3, max_length=100, example="systemV0.1")
     last_ping: datetime = Field(default=None, example=datetime.utcnow())
-    angle_of_view: float = Field(..., gt=0, lt=360)
-    software_hash: str = Field(None, min_length=8, max_length=16)
+    angle_of_view: float = Field(..., gt=0, lt=360, example=10)
+    software_hash: str = Field(None, min_length=8, max_length=16, example="0123456789ABCDEF")
```

Comment:
Just for me to understand, is this linked to the azimuth ?

Reply:
No but I took rthe opportunity.  It os not worth a Pr